test/envtest/reconciler_setupwithmanager_test.go

```diff
@@ -124,7 +161,10 @@ func testNewKonnectEntityReconciler[
 		require.NoError(t, err)
 
 		cl := mgr.GetClient()
-		reconciler := konnect.NewKonnectEntityReconciler[T, TEnt](&ops.MockSDKFactory{}, false, cl)
+		wrapper := ops.NewMockSDKWrapper()
```

Comment:
Can we just refer to this variable as "sdk"? The fact that it's a wrapper on the sdk implementation elsewhere is an implementation detail, right?

Reply:
done.